### PR TITLE
completed task 7

### PR DESCRIPTION
**Pull request by JoeSephian**

### CODE REVIEW

File: __tests__/requests.test.js [hunk 1]
```diff
@@ -108,6 +107,36 @@ describe("getReviews", () => {
   });
 });
 
+describe("postComment", () => {
```
Comment (HannahBurks) [inline __tests__/requests.test.js:110]: Great start, but there are some extra tests we want to be adding here :
Status 201, ignores unnecessary properties
Status 400, invalid ID, e.g. string of "not-an-id"
Status 404, non existent ID, e.g. 0 or 9999
Status 400, missing required field(s), e.g. no username or body properties
Status 404, username does not exist

**Review by HannahBurks — APPROVED** [review]
A nice start Joe, but some further testing and error handling to be added. Once added, please send another PR :)

File: __tests__/requests.test.js (hunk 1 above)
Comment (JoeSephian) [reply __tests__/requests.test.js:110]: Thanks. What is meant by 'ignores unnecessary properties?

Comment (cjplatten) [reply __tests__/requests.test.js:110]: If an addtional property other than author or body is included in the sent comment body then the post should not be impacted.

**Review by HannahBurks — APPROVED** [review]
Lovely - merge away :)

**Pull request merged**